### PR TITLE
Add news item for NumPy 1.20.0 release

### PR DESCRIPTION
**Pull request by rgommers**

### CODE REVIEW

File: content/en/news.md [hunk 1]
```diff
@@ -3,6 +3,20 @@ title: News
 sidebar: false
 ---
 
+### Numpy 1.20.0 release
+
+_Jan 30, 2021_ -- [NumPy 1.20.0](https://numpy.org/devdocs/release/1.20.0-notes.html)
```
Comment (mattip) [inline content/en/news.md:8]:
```suggestion
_Jan 30, 2021_ -- [NumPy 1.20.0](https://numpy.org/doc/stable/release/1.20.0-notes.html)
```

File: content/en/news.md [hunk 2]
```diff
@@ -3,6 +3,20 @@ title: News
...
+is now available. This is the largest NumPy release to date, thanks to 180+
+contributors. The two most exciting new features are:
+- Type annotations for large parts of NumPy, and a new `numpy.typing` submodule
+  containing `ArrayLike` and `DtypeLike` aliases that users and downstream
+  libraries can use when adding type annotations in their own code.
+- Multi-platform SIMD compiler optimizations, with support for x86 (SSE,
+  AVX), ARM64 (Neon), and PowerPC (VSX) instructions. This yieled significant
```
Comment (mattip) [inline content/en/news.md:15]:
```suggestion
  AVX), ARM64 (Neon), and PowerPC (VSX) instructions. This yielded significant
```

**Review by mattip — COMMENTED** [review]
I don't have permissions to merge the PR, but it looks good to me.

**Comment by rgommers** [comment]
Thanks Matti. Wasn't sure I had the right SIMD PRs. 

> I don't have permissions to merge the PR, but it looks good to me.

I'm having a look at that now. Very puzzling, you are on https://github.com/orgs/numpy/teams/numpy-developers/members which has write access to this repo.

**Pull request merged**